src/components/CalendarEventsDialog.vue

```diff
@@ -421,17 +421,24 @@ async function submitNewMeeting() {
 				<h5 v-if="!isOneToOneConversation" class="calendar-meeting__header">
 					{{ t('spreed', 'Attendees') }}
 				</h5>
-				<NcCheckboxRadioSwitch v-model="selectAll" @update:modelValue="toggleAll">
-					{{ inviteLabel }}
-				</NcCheckboxRadioSwitch>
-				<NcButton v-if="!isOneToOneConversation" type="tertiary" @click="isSelectorOpen = true">
-					<template #icon>
-						<IconAccountPlus :size="20" />
-					</template>
-					{{ t('spreed', 'Add attendees') }}
-				</NcButton>
-				<p>{{ attendeeHint }}</p>
-
+				<NcLoadingIcon v-if="!participantsInitialised"
+					class="calendar-meeting--loading"
+					:name="t('spreed', 'Loading participants')" />
+				<p v-else-if="participantsInitialised && participants.length === 0">
+					{{ t('spreed', 'You are the only participant in the conversation') }}
```

Comment:
Could be guests / federated users. Maybe:
```suggestion
					{{ t('spreed', 'No other participants to send invitations to') }}
```